json/src/main/groovy/grails/plugin/json/view/api/JsonView.groovy

```diff
@@ -40,6 +40,17 @@ trait JsonView extends GrailsView {
      * The parent model, if any
      */
     Map parentModel
+
+    /**
+     * The nested view level used in serialization of expand parameter.
+     */
+    int level
```

Comment:
Could you rename these something more specific to expand, as they have quite generic names